Sources/_CryptoExtras/RSA/RSA_boring.swift

```diff
@@ -564,9 +814,79 @@ extension BoringSSLRSAPrivateKey {
             return output
         }
 
+        fileprivate func blindSignature<D: DataProtocol>(for message: D) throws -> _RSA.BlindSigning.BlindSignature {
+            let rsaPrivateKey = CCryptoBoringSSL_EVP_PKEY_get0_RSA(self.pointer)
+            let signatureByteCount = Int(CCryptoBoringSSL_RSA_size(rsaPrivateKey))
+
+            guard message.count == signatureByteCount else {
+                throw CryptoKitError.incorrectParameterSize
+            }
+
+            let messageBytes: ContiguousBytes = message.regions.count == 1 ? message.regions.first! : Array(message)
+
+            let signatureBytes = try Array<UInt8>(unsafeUninitializedCapacity: signatureByteCount) { signatureBufferPtr, signatureBufferCount in
```

Comment:
I know I mentioned this transformation, but I missed that this ends up as a `Data`. We should use `withUnsafeTemporaryAllocation` here.

Reply:
This got thumbs-uped, but not actually done.

Reply:
Sorry, I missed this one. Done.